packages/mobile/src/components/composer-input/ComposerInput.tsx

```diff
@@ -0,0 +1,285 @@
+import { useCallback, useEffect, useRef, useState } from 'react'
+
+import { useAudiusLinkResolver } from '@audius/common/hooks'
+import { splitOnNewline } from '@audius/common/utils'
+import { Platform, TouchableOpacity, View } from 'react-native'
+import type {
+  NativeSyntheticEvent,
+  TextInputKeyPressEventData,
+  TextInputSelectionChangeEventData
+} from 'react-native/types'
+
+import { Flex, IconSend } from '@audius/harmony-native'
+import { Text, TextInput } from 'app/components/core'
+import { env } from 'app/env'
+import { audiusSdk } from 'app/services/sdk/audius-sdk'
+import { makeStyles } from 'app/styles'
+import { spacing } from 'app/styles/spacing'
+import { useThemeColors } from 'app/utils/theme'
+
+import LoadingSpinner from '../loading-spinner/LoadingSpinner'
+
+import type { ComposerInputProps } from './types'
+
+const BACKSPACE_KEY = 'Backspace'
+
+const messages = {
+  sendMessage: 'Send Message',
+  sendMessagePlaceholder: 'Start typing...'
+}
+
+const useStyles = makeStyles(({ spacing, palette, typography }) => ({
```

Comment:
I think we've been generally move away from makeStyles towards using just `style` prop and the theme hook like:
```ts
const { spacing } = useTheme()
```

but I think it's still an open discussion if we want to go 100% that route, so what you have is fine for now

Reply:
yea this was just moving the styles from the previous spot, but I can look into updating this to update things along the way